item-spec/item-spec.md

```diff
@@ -132,31 +132,38 @@ The following types are commonly for assets and are used as key for the Asset Ob
-some cases, selection of a mime type can be ambiguous. For instance, many STAC items have sidecar
-metadata files associated with the core asset (`.tfw`, Landsat 8 MTL files, etc.) that should use
-mime types appropriate for the file - for instance, if it is a plain text file then `text/plain`
-would be appropriate, `text/xml` would be appropriate for cases the metadata is in XML format.
+The media type of an Item can be used by STAC browsers to better determine what to render and display 
+to users searching and browsing the catalog. 
 
-A set of commonly used MIME types for STAC items include the following:
+[Registered](https://www.iana.org/assignments/media-types/media-types.xhtml) Media Types are
+preferred. In cases where custom vendor-specific media types are necessary, they should
+use the `vnd.` prefix. 
 
-| MIME type                      | Description                                                                             |
+STAC Items that have sidecar metadata files associated with a data asset (e.g, `.tfw`, Landsat 8 MTL files) 
+should use media types appropriate for the the metadata file.  For example, if it is a plain text file, then `text/plain`
+would be appropriate; if it is an XML, then `text/xml` is appropriate.
+
+Common STAC Item Media Types:
+
+| Media Type                      | Description                                                                            |
 | ------------------------------ | --------------------------------------------------------------------------------------- |
-| image/geotiff                  | Georeferenced TIFF. _We would like to register image/geotiff as a mime type with IANA._ |
-| image/geotiff+cog              | Cloud Optimized GeoTIFF                                                                 |
-| image/jp2                      | JPEG 2000                                                                               |
-| image/png                      | Visual PNGs (e.g. thumbnails)                                                           |
-| image/jpeg                     | Visual JPEGs (e.g. thumbnails, oblique)                                                 |
-| text/xml                       | XML metadata                                                                            |
-| application/json               | JSON metadata                                                                           |
-| text/plain                     | Plain text metadata                                                                     |
-| application/geo+json           | GeoJSON                                                                                 |
-| application/geopackage+sqlite3 | GeoPackage                                                                              |
+| `image/tiff` or `image/x.geotiff`     | GeoTIFF TIFF file with standardized georeferencing metadata                             |
```

Comment:
I think I'd remove `image/tiff` here and in the next line. I think we really want to know whenever we have a GeoTiff or COG to be able to make use of their "features".

Also, I am wondering how using `x.` complies with this sentence above: `In cases where custom vendor-specific media types are necessary, they should use the `vnd.` prefix. ` For me that sounds like it should be `image/vnd.geotiff`.

Reply:
I would categorize that as the difference in x. and vnd. as between an open "standard"(e.g., COG, even thought there's not standards body-approved RFC) and a proprietary format from a specific vendor (e.g., ESRI Shapefiles, even though it's a mostly-open spec).  In the first case, it would use x.foo, in the second case it should use something like application/vnd.bar.   

The examples I have are Microsoft Office files- "old" .xls files use `application/vnd.ms-excel`, newer .xlsx use `application/vnd.openxmlformats-officedocument.spreadsheetml.sheet`.  Even though they're technically an open standard, they only became that after MSFT fully developed them and then got a standards body to approve them, but for all intents and purposes they're a vendor specific format.

Reply:
I feel it's important not to **force** users into using an unregistered media type, particularly one that I just made up 😬 Maybe the `type` attribute should be an array instead of a single value, so that a provider can specify as many types as they wish? That way, in the future, when `image/geotiff+tiff` and `image/cog+tiff` are IANA-registered, they can be added to the array without breaking existing applications that are expecting `image/tiff` or `image/x.geotiff`